.github/git-hooks/pre-commit

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Prettify all selected files
+ ./node_modules/.bin/prettier --write .
```

Comment:
This file belongs in `.git/hooks/`, maybe an install-hooks command should be made? Or contribution guidelines on how to set up this hook + install prettier dependency?

Reply:
Realistically, the user can just move the file to their `.git/hooks/` for the repo, or do `npm run format` so I'm going to omit this.